backend/packages/Upgrade/src/api/services/ExperimentUserService.ts

```diff
@@ -393,4 +393,11 @@ export class ExperimentUserService {
       await this.individualAssignmentRepository.deleteExperimentsForUserId(userId, assignedExperimentIds);
     }
   }
+
+  public clearDB(): Promise<string> {
+    getConnection().transaction(async (transactionalEntityManager) => {
```

Comment:
Don't you need `await` before getConnection()?

Reply:
Inside we are using await, and it didn't gave me any error on this. Still for safer side we can put await. I will check and update accordingly

Reply:
It won't give error but your http request will send response before the DB cleanup. I think you should not await then as awaiting might lead to request timeout.
But just keep in mind that DB cleanup might not be complete even though you received successful http response.

Reply:
Oh okay got it. Okay, so should I remove `await` from the latest commit?